showcases/angular-showcase/src/app/components/form/textareas/textareas.component.html

```diff
@@ -0,0 +1,28 @@
+<app-form-wrapper
+	[plain]="plain"
+	[model]="ngModel"
+	[control]="formControl.value"
+>
+	<db-textarea
+		label="Plain"
+		placeholder="Placeholder"
+		message="Description"
+		name="input-name"
+		[value]="plain"
+		(change)="plain = $event.target.value"
+	/>
+	<db-textarea
+		label="ngModel"
+		placeholder="Placeholder"
+		message="Description"
+		name="input-name"
+		[(ngModel)]="ngModel"
+	/>
+	<db-textarea
+		label="formControl"
+		placeholder="Placeholder"
+		message="Description"
+		name="input-name"
+		[formControl]="formControl"
+	/>
```

Comment:
```suggestion
	<fieldset>
		<legend>Different form approaches</legend>
		<db-textarea
			label="Plain"
			placeholder="Placeholder"
			message="Description"
			name="input-name"
			[value]="plain"
			(change)="plain = $event.target.value"
		/>
		<db-textarea
			label="ngModel"
			placeholder="Placeholder"
			message="Description"
			name="input-name"
			[(ngModel)]="ngModel"
		/>
		<db-textarea
			label="formControl"
			placeholder="Placeholder"
			message="Description"
			name="input-name"
			[formControl]="formControl"
		/>
	</fieldset>
```

let's even also include form grouping semantics in our examples.